### ISSUE TITLE
NE wall alpha

### ISSUE THREAD

**Issue by IntoTheAbyss356** [Metalit/Replay#42]
The anytweaks bloom as we know makes walls look substantially better. However, even with it on, the rendering doesn't take into account the alpha of the walls for noodle extensions. The alpha makes them more or less opaque, therefore resulting in some maps looking incredibly transparent(ill by roughsketch mapped by Sands) IDK why I'm making this so long but it would be nice for the render to take into account the alpha of walls. you can look at a render of ill and a PC video of ill and immediately see a difference:) thank you, hope to see this added(just a suggestion) 

as I said, no idea why this is so long, thx!

**Comment by IntoTheAbyss356** [comment]
I might add a photo of the render and a photo of PC tomorrow but I gtg to bed so:)

**Comment by IntoTheAbyss356** [comment]
![E94E53A5-BA12-4480-A7EA-1B9C155F57DD](https://user-images.githubusercontent.com/104661523/199250697-270765b7-5676-4f23-8bc2-b24afc084757.png)
![7F0AC130-9648-4C9E-BA5E-19B733BFF454](https://user-images.githubusercontent.com/104661523/199250770-0d9d9151-a745-41a8-95a4-dbc23fbb2740.png)

**Comment by Metalit** [comment]
I believe this is an issue with NE changing the wall material in an attempt to make them look better during normal gameplay. I can see about having the mod disable that when rendering, but I'm not sure if it will be possible

**Comment by IntoTheAbyss356** [comment]
Yeah, the walls change based on alpha and I was thinking that  of disabling it that when rendering it just changes the distortion based on alpha(if that makes sense?). Lol
